tracee/container_tracer.py

```diff
@@ -130,6 +130,10 @@
             "process_vm_writev", "process_vm_readv", "init_module", "finit_module", "delete_module",
             "symlink", "symlinkat", "getdents", "getdents64", "creat", "open", "openat"]
 
+# We always need kprobes for execve[at] so that we capture the new PID namespace 
+essential_syscalls = ["execve", "execveat"]
```

Comment:
I actually think that do_exit is also an essential event as it removes pid namespaces on container exit. It will probably won't cause functionality issues, but we better don't have map "memory leak"

Reply:
That’s a good catch, will fix